packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -480,6 +480,16 @@
             "name": "stepfunctions_previewstatemachine",
             "description": ""
         },
+        {
+            "name": "stepfunctions_openWorkflowStudio",
```

Comment:
Please plan to upstream this to https://github.com/aws/aws-toolkit-common as you near launch.